ff/src/fields/field_hashers/expander/mod.rs

```diff
@@ -2,98 +2,116 @@
 // With some optimisations
 
 use ark_std::vec::Vec;
-use digest::{DynDigest, ExtendableOutput, Update};
+
+use digest::{FixedOutputReset, ExtendableOutput, Update};
```

Comment:
So IIUC `Digest` is a supertrait of `FixedOutputReset` and we just don't need all of supertrait functionality?